tests/xharness/AppRunner.cs

```diff
@@ -43,7 +41,7 @@ class AppRunner
 		public string AppPath;
 		public string Variation;
 		public BuildToolTask BuildTask;
-		public ISimpleListenerFactory ListenerFactory = new SimpleListenerFactory ();
+		public ISimpleListenerFactory ListenerFactory = new SimpleListenerFactory();
```

Comment:
We use the mono style guidelines, you can set those in VSMac:
<img width="1000" alt="Screenshot 2020-03-05 at 08 45 47" src="https://user-images.githubusercontent.com/2190086/75987320-bcf19400-5ebd-11ea-81c3-7ac0d2f10915.png">
```suggestion
		public ISimpleListenerFactory ListenerFactory = new SimpleListenerFactory ();
```